### PR TITLE
updating README.md file with newly added --config option

### PR DESCRIPTION
**Pull request by popindavibe**
Just updating the Readme file from the output of `-h` option.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ file locations match RHEL or Ubuntu.
 Usage: configsnap [options]
 
 Record useful system state information, and compare to previous state if run
-with PHASE containing "post" or "rollback".
+with PHASE containing "post" or "rollback". An optional file,
```
Comment (rhodesn) [inline README.md]:
```suggestion
with PHASE containing "post" or "rollback". A default config file,
```

File: README.md [hunk 2]
```diff
@@ -11,7 +11,9 @@ file locations match RHEL or Ubuntu.
...
+/etc/configsnap/additional.conf, can be provided for extra files, directories
```
Comment (rhodesn) [inline README.md]:
```suggestion
/etc/configsnap/additional.conf, can be customised to include extra files, directories
```

File: README.md [hunk 3]
```diff
@@ -33,6 +35,9 @@ Options:
                         and --phase
   --pre=PRE_SUFFIX      suffix for files captured at previous state, for
                         comparison
+  -c CONFIG, --config=CONFIG
+                        additional config file to use. Setting this will
+                        overwrite default.
```
Comment (rhodesn) [inline README.md]:
```suggestion
                        override default.
```

**Review by rhodesn — CHANGES_REQUESTED** [review]
I popped a couple of suggestions in just to make it clear that the additional.conf file is read by default whether specified on the command line or not.

**Comment by popindavibe** [comment]
All suggestions approved, ready for review/merge.

**Review by rhodesn — APPROVED** [review]

**Comment by rhodesn** [comment]
Awesome, thanks for reviewing Jean.

**Pull request merged**